services/modules/mumble/models.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import unicode_literals
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+
+
+@python_2_unicode_compatible
+class MumbleUser(models.Model):
+    user = models.OneToOneField('auth.User', related_name='mumble', null=True)
```

Comment:
Because I needed to create the field before the data migration can run and as I'm adding a field to an existing model it needs to have a default or null. Once the data migration has run I can probably remove this, but there's still the potential for the database constraints to fail (user had a mumble account but no related authservicesinfo for some strange out of sync state).

It does need to be changed, but it will result in data loss for the reverse migration.